src/components/studies/StudyCard.tsx

```diff
@@ -237,7 +288,9 @@ const StudyCard: FunctionComponent<StudyCardProps> = ({
   return (
     <>
       <Card
-        className={classes.root}
+        className={`${classes.root} ${
```

Comment:
it is fine if you want to leave it but clsx utility makes it easier to conditionally combined classes.
https://www.npmjs.com/package/clsx
it would look something like this `className={clsx(classes.root, 
          [classes.isJustAdded]: isNewlyAddedStudy )}`